helpers/adminDashboardsHelpers.cfm

```diff
@@ -1,5 +1,11 @@
 <cffunction name="renderAdminDashboard" access="public" returntype="string" output="false">
 	<cfscript>
-		return getController().renderViewlet( event="admin.admindashboards.renderDashboard", args=arguments );
+		var userGenerated = isTrue( arguments.userGenerated ?: "" );
```

Comment:
Rather than this, would be cleaner I think with just:
```cfc
<cffunction ...>
    <cfargument name="userGenerated" type="boolean" required="false" default="false" />
```